examples/webgl_morphtargets.html

```diff
@@ -78,60 +78,20 @@
 
 				//
 
-				const params = {
-					influence1: 0,
-					influence2: 0,
-					influence3: 0,
-					influence4: 0,
-					influence5: 0,
-					influence6: 0,
-					influence7: 0,
-					influence8: 0
-				};
-
+				let params = {};
 				const gui = new GUI();
-
 				const folder = gui.addFolder( 'Morph Targets' );
-				folder.add( params, 'influence1', 0, 1 ).step( 0.01 ).onChange( function ( value ) {
-
-					mesh.morphTargetInfluences[ 0 ] = value;
-
-				} );
-				folder.add( params, 'influence2', 0, 1 ).step( 0.01 ).onChange( function ( value ) {
-
-					mesh.morphTargetInfluences[ 1 ] = value;
-
-				} );
-				folder.add( params, 'influence3', 0, 1 ).step( 0.01 ).onChange( function ( value ) {
-
-					mesh.morphTargetInfluences[ 2 ] = value;
-
-				} );
-				folder.add( params, 'influence4', 0, 1 ).step( 0.01 ).onChange( function ( value ) {
+				
+				for (let i = 0; i < 8; i++) {
+					const x = (i).toString();
+					params["influence" + x] = 0
+					folder.add(params, "influence" + x, 0, 1).step(0.01).onChange(function (value) {
```

Comment:
Although this code is more compact, I feel the previous approach is more expressive and easier to understand (especially for new users).

Reply:
The code just has a for loop to  dynamically adds 2 things :

1) The object properties 
2) Its UI element

Nothing fancy

Reply:
I agree with @Mugen87 👍